ShibbolethSettingsForm.inc.php

```diff
@@ -186,6 +194,30 @@ function execute() {
 			'shibbolethAdminUins',
 			trim($this->getData('shibbolethAdminUins'), "\"\';"),
 			'string'
-		);
+                );
+		$this->_plugin->updateSetting(
+			$this->_contextId,
+			'shibbolethOptional',
+			$this->getData('shibbolethOptional'),
+                        'bool'
+                );
+		$this->_plugin->updateSetting(
+			$this->_contextId,
+			'shibbolethOptionalTitle',
+			trim($this->getData('shibbolethOptionalTitle'), "\"\';"),
```

Comment:
Did this `trim` prove necessary in practice, or was it copied from elsewhere?  It seems odd that we would need it.

Reply:
This was another copy. I'll remove it and see how it goes.